test/e2e_test.go

```diff
@@ -176,6 +176,51 @@ func TestSignVerify(t *testing.T) {
 	mustErr(verify(pubKeyPath, imgName, true, map[string]interface{}{"foo": "bar", "baz": "bat"}, ""), t)
 }
 
+func TestSignVerifyKeylessCertChain(t *testing.T) {
```

Comment:
We can remove this, because this test doesn't exercise the newly added code.

Reply:
removed, sorry - was a placeholder for the future work